src/transformers/models/opt/modeling_opt.py

```diff
@@ -492,7 +492,11 @@ def __init__(self, config: OPTConfig):
         else:
             self.project_in = None
 
-        self.layer_norm = None
+        if config.do_layer_norm_before:
```

Comment:
```suggestion
        if config.do_layer_norm_before and config.transformers_version >= 4.20.1:
```
(with correct regex)

Reply:
Hum I think we'd much rather use a config instead ...